src/transaction/log_page_buffer.c

```diff
@@ -3797,6 +3800,19 @@ logpb_flush_all_append_pages (THREAD_ENTRY * thread_p)
     }
   flush_info->num_toflush = 0;
 
+#if defined (SERVER_MODE)
+  if (get_server_type () == SERVER_TYPE_PAGE)
+    {
+      log_lsa saved_lsa = log_Gl.append.get_nxio_lsa ();
+      if (prm_get_bool_value (PRM_ID_ER_LOG_COMMIT_CONFIRM))
+	{
+	  _er_log_debug (ARG_FILE_LINE, "[COMMIT CONFIRM] Send saved LSA=%lld|%d.\n", LSA_AS_ARGS (&saved_lsa));
+	}
+      std::string message (reinterpret_cast < const char *>(&saved_lsa), sizeof (saved_lsa));
+      ps_Gl.push_active_tran_server_request (ps_to_ats_request::SEND_SAVED_LSA, std::move (message));
```

Comment:
I find these names a bit misleading and hard to 'process'. In this case, is the `active_tran_server` the originator or the destination? I reckon it is the destination. Maybe introduce a "direction" particle like to/from in these names? `push_request_to_active_tran_server`?

Reply:
done